text/0793-polymporphic-relations-without-inheritance.md

```diff
@@ -0,0 +1,135 @@
+---
+Stage: Accepted
+Start Date: 
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): 
+RFC PR: https://github.com/emberjs/rfcs/pull/793
+---
+
+<!--- 
+Directions for above: 
+
+Stage: Leave as is
+Start Date: 2022-02-11
+Release Date: Leave as is
+Release Versions: Leave as is
+Relevant Team(s): Ember Data
+RFC PR: https://github.com/emberjs/rfcs/pull/793
+-->
+
+# Polymorphic relations without inheritance
+
+## Summary
+
+Allow setting polymorph relations without inheritance and mixins.
+
+## Motivation
+
+In pre-octane Ember you could set polymorphic relations using two ways: inheritance or mixins. Mixins are deprecated and impossible to use with native classes, so the only option is to use inheritance:
```

Comment:
Mixins aren't officially deprecated so lets not say this. They can be used with native classes via either a decorator approach or a double-extend ala 

```ts
export default class Post extends Model.extend(TaggableMixin) {

}
```

They are discouraged and we do want to simplify the programming model though. The motivation here is that polymorphic relationships should be possible without mixins or inheritance.